backend-tests/tests/pytest.ini

```diff
@@ -3,3 +3,5 @@
 # We keep it in xunit1 until we align the tooling between test frameworks
 [pytest]
 junit_family = xunit1
+markers =
+    storage_test: marks tests that uses artifact storage
```

Comment:
The only inconvenient that I see with this design is that we need to add the marker as we add tests in the future.

Another way to engineer this can be to have a marker for tests that need to be skept for Azure storage (similar as we skip tests when running in K8S). And with this way you don't risk missing tests (you only risk that your new test fails because cannot be run in such setup, and they you just mark it so that it gets skept).

Reply:
Let's go with opt-in. You (backend team) are the owners of this test suite, so let's do it as you think is best!